pennylane/qnn/keras.py

```diff
@@ -0,0 +1,292 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module is for PennyLane-Keras integration."""
+import functools
+import inspect
+from collections.abc import Iterable
+from typing import Optional
+
+try:
+    import tensorflow as tf
+    from tensorflow.keras.layers import Layer
+    from pennylane.interfaces.tf import to_tf
+
+    CORRECT_TF_VERSION = int(tf.__version__.split(".")[0]) > 1
+except ImportError:
+    from abc import ABC
+
+    Layer = ABC
+    CORRECT_TF_VERSION = False
```

Comment:
Might as well leave a comment here with the answer to Nathan's question earlier, so future devs understand what is happening here.
```suggestion
    # If TensorFlow 2.0 is not installed, import a 'mock' Layer class. This allows
    # The KerasLayer to be imported. Instead, an error will be raised on initialization of
    # the KerasLayer if CORRECT_TF_VERSION=False.
    Layer = ABC
    CORRECT_TF_VERSION = False
```

Reply:
Ah, the comment needs to be rewritten likely!

Essentially, this allows the module to be imported even though TF might not be installed. The 'ImportError' is instead moved to the class initialization point.